src/js/edu-benefits/5490/config/form.js

```diff
@@ -498,11 +502,25 @@ const formConfig = {
                   'ui:options': {
                     hideIf: form => {
                       const status = _.get('highSchool.status', form);
-                      return status !== 'graduated';
+                      return status !== 'graduated' && status !== 'graduationExpected';
                     },
-                    expandUnder: 'status'
-                  }
-                }
+                    expandUnder: 'status',
```

Comment:
We probably need to add a way to use expandUnder for specific values, instead of just checking if it's truthy. That would make this work better.